yarn.lock

```diff
@@ -26723,6 +26730,11 @@ typescript@4.3.5, typescript@~4.3.2:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.3.5.tgz#4d1c37cc16e893973c45a06886b7113234f119f4"
   integrity sha512-DqQgihaQ9cUrskJo9kIyW/+g0Vxsk8cDtZ52a3NGh0YNTfpUSArXSohyUGnvbPazEPLu398C0UxmKSOrPumUzA==
 
+typescript@^4.8.2:
```

Comment:
**chore:** we use only 1 single ts version as ts by itself is unable to work with multiple ones. until resolved this is one of main showstoppers.